src/06-loops.js

```diff
@@ -25,12 +31,21 @@ function rangeSum(min, max) {}
  *  rangeOdd(10, 15);
  *  //> [ 15, 13, 11 ]
  */
-function rangeOdd(min, max) {}
+function rangeOdd(min, max) {
+  let myArray = [];
+  for (i = max; i >= min; --i){
+    if(i%2 !== 0){
```

Comment:
Let's also add some spacing to our modulus operators ` i % 2 ` is easier to read.